src/balancerd/src/dyncfgs.rs

```diff
@@ -23,7 +23,16 @@ pub const SIGTERM_WAIT: Config<Duration> = Config::new(
     "Duration to wait after SIGTERM for outstanding connections to complete.",
 );
 
+/// Whether to log the status of pgwire connections.
+pub const LOG_PGWIRE_CONNECTION_STATUS: Config<bool> = Config::new(
+    "balancerd_log_pgwire_connection_status",
+    false,
+    "Whether to log the status of pgwire connections.",
+);
```

Comment:
Sorry to keep going back and forth on this, but I think we should add a `balancerd_log_filter` parameter here, and wire it up to the tracing handle the same way we do in `environmentd`.

Reply:
(Let me know if you want help with this; the tracing stuff is a bit of a labyrinth; hopefully this is easy to wire up with the `TracingHandle` though.)

Reply:
Do you think we need any of the other tracing configs like:

- opentelemetry_filter
- log_filter_defaults
- opentelemetry_filter_defaults
- sentry_filters

?

Reply:
I think the only one we need as an end user to track this escalation down is `log_filter`. I wouldn't bring any of the others in for their own sake, but if it turns out that it's easier to plumb all of these as a bundle, then I'd say go for it.

Reply:
It's also not clear to me how we'd update the `TracingHandle` whenever the parameter changes. I guess we'd have to plumb the handle into the task that syncs the dyncfg and update it there.

Reply:
Precisely. Here's how compute does it, for example:

https://github.com/MaterializeInc/materialize/blob/4de9be0d4178aab942caff3a345f5052fcb077eb/src/compute/src/compute_state.rs#L374

So you may find it's easier to have al the parameters for a `TracingParameters`, and then you can do `tracing_parameters.apply(tracing_handle)` in the dyncfg sync loop.

Reply:
OK sounds good, I'm going to do this in a separate PR because it feels isolated enough: https://github.com/MaterializeInc/materialize/pull/28905.